core/ustring.cpp

```diff
@@ -3296,7 +3296,7 @@ String String::humanize_size(size_t p_size) {
 	int digits = prefix_idx > 0 ? _humanize_digits(p_size / _div) : 0;
 	double divisor = prefix_idx > 0 ? _div : 1;
 
-	return String::num(p_size / divisor).pad_decimals(digits) + prefix[prefix_idx];
+	return String::num(p_size / divisor).pad_decimals(digits) + RTR(prefix[prefix_idx]);
```

Comment:
That's bogus, `RTR()` and `TTR()` strings are extracted, not compiled, and there's no string to translate in `prefix[prefix_idx]`. It's on the original strings themselves that you should use `RTR` (and the leading space should likely be left out).